modules/weather.py

```diff
@@ -17,8 +17,14 @@
 
 
 def location(q):
-    uri = 'https://nominatim.openstreetmap.org/search/?q={query}&format=json'.\
-        format(query=web.quote(q))
+    uri = 'https://nominatim.openstreetmap.org/search?%s={query}&format=json'
```

Comment:
I like this change, but can you please use `format` instead of the the old `%` way?